libtelemetry/src/androidTestFull/java/com/mapbox/android/telemetry/location/LocationCollectionClientInstrumentedTest.java

```diff
@@ -0,0 +1,70 @@
+package com.mapbox.android.telemetry.location;
+
+import android.os.MessageQueue;
+import android.support.test.InstrumentationRegistry;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class LocationCollectionClientInstrumentedTest {
+  private LocationCollectionClient ref;
+
+  @Before
+  public void setUp() {
+    ref = LocationCollectionClient.install(InstrumentationRegistry.getTargetContext(),
+      1000);
+  }
+
+  @After
+  public void tearDown() {
```

Comment:
Since it is a Singleton, `ref = null` will do nothing to it. We should call `uninstall` in this method.

Reply:
yeah, good catch thanks!